crates/uv/tests/pip_compile_scenarios.rs

```diff
@@ -27,9 +27,9 @@ fn command(context: &TestContext, python_versions: &[&str]) -> Command {
         .arg("compile")
         .arg("requirements.in")
         .arg("--index-url")
-        .arg("https://test.pypi.org/simple")
+        .arg("https://astral-sh.github.io/packse/0.3.6/simple-html/")
```

Comment:
This is a major change — no more reliance on PyPI itself (which also means we can be spec-incompliant)

Reply:
We also have the capability to compile the static index once at the start of a test run. I'd want to do some more work to make sure that's fast and such but it could be more convenient than requiring a packse release to add a new scenario.